### ISSUE TITLE
cygwin build for cygwin trys to install to //usr/lib

### ISSUE THREAD

**Issue by djhenderson** [capstone-engine/capstone#768]
I am building capstone next branch using cygwin, for install and use under cygwin. My intention is to use capstone with  python and with other software that has *NIX only versions. The install fails as in:

```
$ ./make.sh install
mkdir -p //usr/lib
mkdir: cannot create directory ‘//usr’: Read-only file system
make: *** [Makefile:410: install] Error 1
```

Supplying the default PREFIX, as in 

```
$ PREFIX=/usr ./make.sh install
```

also fails, but

```
$ PREFIX=usr ./make.sh install
install -m0755 ./capstone. /usr/lib
install -m0644 ./capstone.lib /usr/lib
mkdir -p /usr/include/capstone
install -m0644 include/capstone/*.h /usr/include/capstone
mkdir -p /usr/lib/pkgconfig
install -m0644 ./capstone.pc /usr/lib/pkgconfig
```

partially works, but the library name is missing the .dll extension. Using the master branch, on Aug 28, I was able to install with PREFIX=~ and with no prefix as evidenced by:

$ ls -l ~/lib/caps*
-rwxr-xr-x  1 Doug None 2.6M Aug 28 22:03 capstone.dll*
-rw-r--r--  1 Doug None 3.1M Aug 28 22:03 capstone.lib

$ ls -l /usr/lib/caps*
-rwxr-xr-x 1 Doug None 3.8M Sep  5 12:19 /usr/lib/capstone.*
-rwxr-xr-x 1 Doug None 2.6M Aug 28 23:09 /usr/lib/capstone.dll*
-rw-r--r-- 1 Doug None 4.3M Sep  5 12:19 /usr/lib/capstone.lib

On my previous install, I omitted building for all but x86 achitecture, today I am building for all archs.

Uninstall also exhibits the problem with PREFIX, and it also has incorrect file names.

To summarize, 
- make.sh install prepends a / to the PREFIX path. On windows, I think, a leading double slash takes you to a directory containing your computer and other computers on your network. As I recall, extra leading slashes are ignored on *NIX systems.
- library file names and extensions are incorrect for a build on cygwin for install on cygwin.
- make.sh uninstall prepends a / to the PREFIX path, and has library file name problems

**Comment by aquynh** [comment]
can you propose a fix for this?

**Comment by kabeor** [comment]
Close this issue for now because of legacy, we are preparing to release Capstone 5.0, plz feel free to open a new issue if that still has this issue.
thx :)
